### PR TITLE
LPS-171427 - RemoveClickInMacro

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-171427

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hey @JhossephG,

You need to use source format to fix it and the commit name being as _**LRQA-XXXXX** Remove the 'Button.clickSave' to the macro 'addCustomFieldEntryViaUI'_.

File: modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro [hunk 1]
```diff
@@ -8,8 +8,6 @@ definition {
 			locator1 = "ObjectPortlet#ENTRY_FIELD_GENERIC",
 			value1 = "${entry}");
 
-		Button.clickSave();
-
 		WaitForSPARefresh();
 
 		WaitForPageLoad();
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro:13]: Hey,

You could create a new macro for this case and call it separated on testcase. Doesn't make sense to remove only Button.clickSave(), the new macro can contain the three lines (Button.clickSave, WaitForSPARefresh and 
WaitForPageLoad).

So, you can remove the lines on **addCustomFieldEntryViaUI** and call it directly on testcase.

Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/macros/ObjectPortlet.macro:13]: If you want, I may explain it better to you.

**Comment by andreefarias** [comment]
Put the tickets in the description.

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by JhossephG** [comment]
ci:test:sf

**Review by andreefarias — APPROVED** [review]

**Comment by andreefarias** [comment]
Great! Look good to me.

**Comment by andreefarias** [comment]
ci:forward

**Comment by timpak** [comment]
Hey @andreefarias and @JhossephG looks like this was never forwarded. Do we still need this? If so, we should fix the merge conflicts and forward it.

Also, instead of creating a new macro, can we use PortletEntry.save(); instead?

**Comment by JhossephG** [comment]
> Hey @andreefarias and @JhossephG looks like this was never forwarded. Do we still need this? If so, we should fix the merge conflicts and forward it.
> 
> Also, instead of creating a new macro, can we use PortletEntry.save(); instead?

It's true. I'll go take a look on this later and change this things.

**Comment by andreefarias** [comment]
ci:close

**Pull request closed without merging**